src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -4,9 +4,9 @@
 
 use ArrayAccess;
 use Closure;
+use GrahamCampbell\TestBenchCore\Assert as PHPUnit;
```

Comment:
Doesn't this now make it a requirement?

Reply:
I'd say no. We do the same with Mockery in loads of places, but we don't make it a requirement.

<img width="401" alt="image" src="https://user-images.githubusercontent.com/2829600/71580004-c98cf100-2af6-11ea-831f-5b1392ccd06f.png">

Reply:
In fact, we do the same for all our optional dependencies. We use them, and then stuff breaks if they are not installed.

Reply:
But surely the amount of usages of the `Assert` class will break things for people who forget to require the package?

Reply:
Yeh. The same will happen if they forget to require mockery though?

Reply:
Mockery is a dependency of laravel/laravel so it's unlikely that people don't have that.

Reply:
Yeh, I'm planning on making testbench-core a dev-dependency of laravel/laravel. ;)

Reply:
https://github.com/laravel/laravel/pull/5194

Reply:
This changes is actually slightly different from other deps.

From my understanding the requirement for testbench-core was merely for array subset. However with this even `PHPUnit::assertTrue()` now also depends on testbench-core.